src/LiveComponent/src/LiveDownloadResponse.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Symfony\UX\LiveComponent;
+
+use SplFileInfo;
+use SplTempFileObject;
+use Symfony\Component\HttpFoundation\BinaryFileResponse;
+use Symfony\Component\HttpFoundation\HeaderUtils;
+
+/**
+ * @author Simon André <smn.andre@gmail.com>
+ */
+final class LiveDownloadResponse extends BinaryFileResponse
+{
+    public const HEADER_LIVE_DOWNLOAD = 'X-Live-Download';
+
+    public function __construct(string|SplFileInfo $file, ?string $filename = null)
```

Comment:
What about adding `\Symfony\Component\HttpFoundation\File\File` as type of `$file`? 
Even if `File` already extends `SplFileInfo`, it may be interesting to add it IMO

Reply:
Well, I see that `BinaryFileResponse::__construct()` _only_ accepts `\SplFileInfo|string`, let's keep it as it is.

Reply:
What would be the goal to add File ?

Reply:
Nothing except documentation/DX, when I see `string|`SplFileInfo`, I don't really know I can pass a `Symfony\Component\HttpFoundation\File\File` (except if I know its parent class)

But that's fine, let's keep aligned to `BinaryFileResponse` constructor.